client/query.go

```diff
@@ -62,35 +62,27 @@ func (c *QueryClient) Query(nrqlQuery string, response interface{}) (err error)
 		return errors.New("go-insights: Invalid query response can not be nil")
 	}
 
-	err = c.generateQueryURL(nrqlQuery)
-	if err != nil {
-		return err
-	}
-
-	c.Logger.Debugf("Querying: %s", nrqlQuery)
-	err = c.queryRequest(response)
-	if err != nil {
-		return err
-	}
+	err = c.queryRequest(nrqlQuery, response)
```

Comment:
Need to check if `queryRequest` returned an error, and return that instead of the explicit `nil`.

Reply:
Checking and returning err here now.